kahuna/public/js/crop/controller.js

```diff
@@ -6,14 +6,18 @@ var crop = angular.module('kahuna.crop.controller', ['gr.keyboardShortcut']);
 
 crop.controller('ImageCropCtrl',
                 ['$scope', '$rootScope', '$stateParams', '$state',
-                 '$filter', '$document', 'mediaApi', 'mediaCropper',
+                 '$filter', '$document', '$window', 'mediaApi', 'mediaCropper',
                  'image', 'optimisedImageUri', 'keyboardShortcut',
                  function($scope, $rootScope, $stateParams, $state,
-                          $filter, $document, mediaApi, mediaCropper,
+                          $filter, $document, $window, mediaApi, mediaCropper,
                           image, optimisedImageUri, keyboardShortcut) {
 
     const ctrl = this;
     const imageId = $stateParams.imageId;
+    if ($stateParams.cropType) {
+        $window.sessionStorage.setItem('cropType', $stateParams.cropType);
```

Comment:
can we use [storage.js](https://github.com/guardian/grid/blob/master/kahuna/public/js/util/storage.js) for this?